ITC503/ITC503-IOC-01App/Db/ITC503.db

```diff
@@ -0,0 +1,603 @@
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:TEMP")
+  field(SDIS, "$(P)DISABLE")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(ai, "$(P)TEMP2") {
+  field(DESC, "Get sensor 2 temp")
+  field(SCAN, "Passive")
+  field(DTYP, "stream")
+  field(INP, "@itc503.proto get_s2temp $(PORT)")
+  field(PREC, "4")
+  field(EGU, "K")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:TEMP")
+  field(SDIS, "$(P)DISABLE")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(ai, "$(P)TEMP3") {
+  field(DESC, "Get sensor 3 temp")
+  field(SCAN, "Passive")
+  field(DTYP, "stream")
+  field(INP, "@itc503.proto get_s3temp $(PORT)")
+  field(PREC, "4")
+  field(EGU, "K")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:TEMP")
+  field(SDIS, "$(P)DISABLE")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bo, "$(P)RAMP:STATUS:SP")
+{
+   field(DESC, "Toggles the SP ramp")
+   field(DTYP, "asynInt32")
+   field(OUT,  "@asyn($(READ),0,1)RMP")
+   field(ZNAM, "OFF")
+   field(ONAM, "ON")
+}
+
+record(bi, "$(P)RAMP:STATUS")
+{
+   field(DESC, "Toggles the SP ramp")
+   field(DTYP, "asynInt32")
+   field(INP,  "@asyn($(READ),0,1)RMP")
+   field(SCAN, "I/O Intr")
+   field(ZNAM, "OFF")
+   field(ONAM, "ON")
+}
+
+record(ao, "$(P)RAMP:RATE:SP")
+{
+   field(DESC, "Rate the ramp increases/decreases")
+   field(DTYP, "asynFloat64")
+   field(OUT, "@asyn($(READ),0,1)RATE")
+   field(EGU, "K/min")
+}
+
+record(ai, "$(P)RAMP:RATE")
+{
+   field(DESC, "Rate the ramp increases/decreases")
+   field(DTYP, "asynFloat64")
+   field(INP, "@asyn($(READ),0,1)RATE")
+   field(EGU, "K/min")
+   field(SCAN, "I/O Intr")
+   info(INTEREST, "MEDIUM")
+}
+
+record(ao, "$(P)TTEMP:SP") {
+  field(DESC, "Set target temp")
+  field(SCAN, "Passive")
+  field(PREC, "2")
+  field(EGU, "K")
+  field(DTYP, "asynFloat64")
+  field(OUT, "@asyn($(READ),0,1)TGT")
+  
+  # Readascii ramp is responsible for setting TTEMP:SP:RAW, but have to trigger it
+  field(FLNK, "$(P)RAMP:_OUT_SP.PROC CA")
+}
+
+record(ai, "$(P)RAMP:_OUT_SP")
+{
+   field(DESC, "The value that the ramp is setting")
+   field(DTYP, "asynFloat64")
+   field(INP,  "@asyn($(READ),0,1)SP")
+   field(SCAN, "I/O Intr")
+   field(FLNK, "$(P)TTEMP:SP:RAW.PROC CA")
+}
+
+# This is the value that is being sent to the ReadASCII ramp to tell it at what temperature we currently are
+record(ao, "$(P)RAMP:_CURRENT_TTEMP")
+{
+    field(DESC, "The current temperature of the sensor")
+	field(DOL, "$(P)TTEMP")
+	field(OMSL, "closed_loop")
+	field(DTYP, "asynFloat64")
+	field(OUT, "@asyn($(READ),0,1)CUR")
+}
+
+record(ao, "$(P)TTEMP:SP:RAW") {
+  field(DESC, "Set target temp")
+  field(SCAN, "Passive")
+  field(DTYP, "stream")
+  field(DOL, "$(P)RAMP:_OUT_SP")
+  field(OUT, "@itc503.proto set_ttemp $(PORT)")
+  field(OMSL, "closed_loop")
+  field(PREC, "2")
+  field(EGU, "K")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:TTEMP:SP")
+  field(SDIS, "$(P)DISABLE")
+}
+
+record(ai, "$(P)TTEMP") {
```

Comment:
This is surely `TEMP:SP:RBV`? Assuming we don't want to confuse people by calling it `TEMP` because that is covered by the 3 other sensors.